### PR TITLE
Add `active` index for table `at_target_groups`

### PR DESCRIPTION
**Pull request by ghost**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| License | MIT

#### What's in this PR?

New index for the `active` field in the `at_target_groups` table.

#### Why?

Query optimization.

##### Before
image

##### After
image

#### SQL Migration

```sql
CREATE INDEX active_idx ON `at_target_groups` (`active`)
```

### CODE REVIEW

File: src/Sulu/Bundle/AudienceTargetingBundle/Resources/config/doctrine/TargetGroup.orm.xml [hunk 1]
```diff
@@ -35,5 +35,9 @@
                 <cascade-persist/>
             </cascade>
         </one-to-many>
+
+        <indexes>
+            <index name="active_idx" columns="active"/>
+        </indexes>
```
Comment (alexander-schranz) [inline src/Sulu/Bundle/AudienceTargetingBundle/Resources/config/doctrine/TargetGroup.orm.xml]: Thx, can you move it to the top like here: https://github.com/sulu/sulu/blob/8ed3425211e8b72b4ac7ecc5239842408927fc42/src/Sulu/Bundle/ContactBundle/Resources/config/doctrine/Contact.orm.xml#L8-L11

And we would require a line in the `UPGRADE.md` about the SQL changes.

**Review by alexander-schranz — CHANGES_REQUESTED** [review]
@flogado Thx for the pull request. 

This should be targeting the `2.4` branch can you rebase it. I changed the target branch.

File: src/Sulu/Bundle/AudienceTargetingBundle/Resources/config/doctrine/TargetGroup.orm.xml (hunk 1 above)
Comment (ghost) [reply src/Sulu/Bundle/AudienceTargetingBundle/Resources/config/doctrine/TargetGroup.orm.xml]: Done. :)

**Comment by alexander-schranz** [comment]
@flogado Thank you!

**Pull request merged**